src/collective/recaptchacontactform/__init__.py

```diff
@@ -9,47 +9,47 @@
 from zope.component import getMultiAdapter
 from z3c.form import button
 
+import inspect
 import Products.CMFPlone.browser.contact_info
 
 _ = MessageFactory('collective.recaptchacontactform')
 
+if AutoExtensibleForm not in inspect.getmro(ContactForm):
```

Comment:
@sneridagh just out of curiosity. Why don't we just check for the IAutoExtensibleForm interface?